pkgs/development/libraries/qrcodegen/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv
+, fetchFromGitHub
+, lib
```

Drop this, we'll use `stdenv.lib` below

```suggestion
, lib
```